app/models/miq_report/generator/async.rb

```diff
@@ -54,7 +54,7 @@ def async_generate_table(options = {})
     unless sync # Only queued if sync reporting disabled (default)
       cb = {:class_name => task.class.name, :instance_id => task.id, :method_name => :queue_callback_on_exceptions, :args => ['Finished']}
       unless self.new_record?
-        MiqQueue.put(
+        MiqQueue.put_with_queue(
```

Comment:
does it make sense to get cb into here?
not sure if that is useful in the table

(aah, reuse)